.github/actions/patch-kernel/action.yml

```diff
@@ -0,0 +1,9 @@
+name: 'patch kernel tree'
+description: 'patch kernel'
+runs:
+  using: "composite"
+  steps:
+    - name: patch bpf-next tree
```

Comment:
we patch bpf tree as well? So maybe keep more generic: "apply temporary patches"?